docs/configure/styling-and-css.md

```diff
@@ -28,8 +28,15 @@ If your component files import their CSS, Storybook’s webpack config will work
 
 <!-- prettier-ignore-end -->
 
-To use your CSS in all stories, you import it in [`.storybook/preview.js`](./overview.md#configure-story-rendering)
+To use your (S)CSS in all stories, you import it to the styles list of your `build-storybook` configuration like this: 
+```json
+    "build-storybook": {
+      "options": {
+        "styles": ["apps/my-default-app/src/styles.scss"]
```

Comment:
Just some information. I don't know if it is interesting. I don't even know if it's necessary to add it here. but FYI

If the user uses the `browserTarget` option without the `styles` option then it will use the styles of the target.
If both are given the style defined here will be used.

In some cases we will want to add a main scss file only for Storybook. For example if we have one Storybook for several projects and applications inside angular workspace
Ex : 
```suggestion
        "styles": [".storybook/styles.scss"]
```

Reply:
> If the user uses the browserTarget option without the styles option then it will use the styles of the target.
If both are given the style defined here will be used.

This does not seem to work with Nx and Angular both version 13.x. Have just tired with a production project. It looks like you really need to tell storybook directly which styles to use.